### PR TITLE
Handle cases when <Tr>s or <Td>s are null

### LINKED ISSUE #148: Providing `null` as a column throws an error
For example, when only displaying a column conditionally via a ternary

### PR DESCRIPTION
**Pull request by EpicH0liday**
When looping through rows and columns, this simply returns if the row
or column is null. No warning is given, because this is a valid use
case where the row or column in question might need to be blank,
whereas before this commit you would have had to render an empty row or
column

Fixes #148

### CODE REVIEW

File: src/reactable.jsx [hunk 1]
```diff
@@ -519,30 +519,30 @@
                     React.Children.forEach(child.props.children, function(descendant) {
                         // TODO
                         /* if (descendant.type.ConvenienceConstructor === Td) { */
-                        if (true) {
```
Comment (EpicH0liday) [inline src/reactable.jsx]: this is an `if (true)` that doesn't have an else case, is it actually being used for anything??

Comment (glittershark) [inline src/reactable.jsx]: See the commented out TODO above

Comment (EpicH0liday) [inline src/reactable.jsx]: I figured that was it, but I feel weird about having an `if (true)` just as a placeholder. your call tho

Comment (glittershark) [inline src/reactable.jsx]: Bonus points if you submit a separate PR DO-ing the TODO :wink:

File: build/reactable.js [hunk 2]
```diff
@@ -510,7 +510,9 @@
                        return; // (continue)
                        }
                        */
-                    if (typeof(child.props) !== 'object') {
+                    if (child == null) {
```
Comment (glittershark) [inline build/reactable.js]: Let's do the same thing you do on line 524 here

Comment (EpicH0liday) [inline build/reactable.js]: since that's kindof what's being done on the line following this, should i just collapse that into one `if` and remove the `console.warn`?

Comment (glittershark) [inline build/reactable.js]: :+1: sounds good to me.
